rosidl_runtime_py/get_interfaces.py

```diff
@@ -171,6 +171,8 @@ def get_interface_path(interface_name: str) -> str:
             f"Invalid name '{interface_name}'. Expected at least two parts separated by '/'")
     if not all(parts):
         raise ValueError(f"Invalid name '{interface_name}'. Must not contain empty parts")
+    if '..' in parts:
+        raise ValueError(f"Must not contain '..'")
```

Comment:
Either include `Invalid name '{interface_name}'.` in the message or remove the leading `f` since the string contains no placeholders anymore. The former seems to provide more context from a user point of view.

Reply:
You're right. This makes more sense. I've updated the PR.